documentation/modules/proc-deploying-auth-services.adoc

```diff
@@ -40,22 +40,22 @@ endif::[]
 
 .Prerequisites
 
-* Modify the installation files according to the namespace the Standard Authservice is going to be installed in
+* (Optional) Replace namespace if you want to deploy to something other than `{ProductNamespace}` (and substitute `{ProductNamespace}` in subsequent steps):
 +
-[source, subs="+quotes"]
+[options="nowrap",subs="+quotes,attributes"]
 ----
-sed -i 's/myproject/_<my-namespace>_/' install/components/standard-authservice/*RoleBinding*.yaml
+sed -i 's/{ProductNamespace}/_<my-namespace>_/' install/components/standard-authservice/\*RoleBinding*.yaml
 ----
 
 .Procedure
 
 ifeval::["{cmdcli}" == "kubectl"]
 . Create a certificate to use with the `standard` authentication service. For testing purposes, you can create a self-signed certificate:
 +
-[options="nowrap"]
+[options="nowrap",subs="+quotes,attributes"]
 ----
 mkdir -p standard-authservice-cert
-openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=standard-authservice.enmasse.svc.cluster.local" -out standard-authservice-cert/tls.crt -keyout standard-authservice-cert/tls.key
+openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=standard-authservice.{ProductNamespace}.svc.cluster.local" -out standard-authservice-cert/tls.crt -keyout standard-authservice-cert/tls.key
 ----
```

Comment:
<{ProductNamespace}>

Add:
"where <{ProductNamespace}> is the namespace where {ProductName} was deployed."